bot/discord/commands/staff/pollping.js

```diff
@@ -0,0 +1,12 @@
+const Discord = require("discord.js");
+exports.run = (client, message, args) => {
+    // Check if user has the dev role
+    if (!message.member.roles.cache.find((r) => r.id === "898041747597295667")) return;
+    // Check the message is in the #dev-questions channel
+    if(message.channel.id !== "1083142107977486389") return message.reply("This command can only be used in <#1083142107977486389>!");
+    // Command can only be used every 30 minutes
+    if(pollPingLastUsed + 1800000 > Date.now()) return message.reply(`This command can only be used every 30 minutes! Cooldown expires <t:${(pollPingLastUsed + 1800000).toString().slice(0, -3)}:R>`)
```

Comment:
nit: can we please use prettier to not have 1000 characters per line? This will eventually end up being painful to read.

Reply:
Fixed